src/types.h

```diff
@@ -254,7 +254,7 @@ class SizedType
   };
   bool IsTupleTy(void) const
   {
-    return type == Type::tuple;
+    return type == Type::tuple || type == Type::timestamp;
```

Comment:
timestamp is not a tuple type. I don't think you can do this. Why was this necessary?